src/Resources/translations/SonataMediaBundle.sl.xliff

```diff
@@ -483,10 +483,6 @@
         <source>form.label_start_paused</source>
         <target>Začetek pavze</target>
       </trans-unit>
-      <trans-unit id="form.label_wrap">
```

Comment:
It is removed because it is not available in en?

Reply:
Yes